Week1/database.js

```diff
@@ -0,0 +1,93 @@
+createQuery(create_country);
+
+const create_city =
+  'CREATE TABLE city (ID INT NOT NULL AUTO_INCREMENT,Name VARCHAR(60) NOT NULL, CountryCode INT NOT NULL, District VARCHAR(60), Population INT NOT NULL , PRIMARY KEY (ID) )';
+createQuery(create_city);
+
+const insert_country_queries = [
+  "INSERT INTO country VALUES ('Netherlands', 'Europe', 'Western Europe', 42508 , 1581, 17.08, 81.51, 901900000, NULL, 'Holland', 'Constitutional Monarchy', 'King Willem-Alexander', 'Amsterdam')",
+  "INSERT INTO country VALUES ('Turkey', 'Asia', 'Europe', 783562 ,1923, 79.81 ,75.76, 2225000000, NULL,'Türkiye','Republic', 'Recep Tayyip Erdogan','Ankara')",
+  "INSERT INTO country VALUES ('Afghanistan', 'Asia', 'Southern and Central Asia', 652237, 1919, 35.53, 63.67, 71050000, NULL, 'Islamic Emirate of Afghanistan', 'Islamic republic','Ashraf Ghani', 'Kabul')",
+  "INSERT INTO country VALUES ('Canada', 'North America', 'North America', 9985000, 1867, 37.06,82.30, 1679000000,NULL, 'Canada', 'Constitutional Monarchy', 'Queen Elizabeth II','Ottowa')",
+  "INSERT INTO country VALUES ('Germany', 'Europe', 'Europe',357386, 1990, 82.79, 80.64, 4280000000 , NULL, 'Deutschland', 'Republic','Frank-Walter Steinmeier', 'Berlin')",
+];
+insertValues(insert_country_queries);
+
+const insert_city_queries = [
+  "INSERT INTO city SET name='Amsterdam', CountryCode=31, District='North Holland', Population= 821752",
+  "INSERT INTO city SET name='Rotterdam', CountryCode=31, District='South Holland', Population= 623652",
+  "INSERT INTO city SET name='Utrecht', CountryCode=31,District= 'Utrecht',Population= 1262000",
+  "INSERT INTO city SET name='Izmir',CountryCode=90,District='Aegean',Population= 4224000",
+  "INSERT INTO city SET name='Berlin',CountryCode=49,District='Berlin',Population= 3575000",
+];
+insertValues(insert_city_queries);
+
+const countryPopulationOver8million =
+  'SELECT name AS country_population_over_8_million FROM country WHERE Population > 8000000';
+createQuery(countryPopulationOver8million);
+
+const countryHaveLand =
+  "SELECT name AS country_includes_land FROM country WHERE name LIKE '%land%'";
+createQuery(countryHaveLand);
+
+const citiesPopulationBetween =
+  'SELECT name AS cities_between_5billion_and_1million FROM city WHERE Population BETWEEN 500000 AND 1000000';
+createQuery(citiesPopulationBetween);
+
+const continentsInEurope =
+  "SELECT name AS continents_in_Europe FROM country WHERE Continent='Europe'";
+createQuery(continentsInEurope);
+
+const countrySurfaceAreaDesc =
+  'SELECT name,SurfaceArea AS countries_surfaceArea_in_desc FROM country ORDER BY SurfaceArea DESC';
+createQuery(countrySurfaceAreaDesc);
+
+const citiesInNetherlands =

```

Comment:
It's nice you tried the Join !
It this case it's not really useful because you still have no "connection" between city table and country table. It will come next week with the foreign key.
Instead of the JOIN you could also have select on both table city and country done a WHERE clause with condition on both table. Want to try ?

Reply:
I tried JOIN because I could not find another solution. I did not find any similarities between city and country tables especially to get cities in Netherlands in order to match and use condition in query. Can you give me a hint:)

Reply:
The Join is absolutely fine don't worry, it's just an other way of doing it that I find maybe more readable for this query but an other developer could have an other opinion.

You can select on multiple table and use condition to retrieve the correct rows .
Example with two tables mytable1 and mytable2:
SELECT mytable1.columA,mytable2.columnB FROM mytable1,mytable2 WHERE mytable1.columnC = 'something' AND mytable2.columnD = 'something else';
Actually it's even called implicit join (I had to google it :) )

So here you can simply select from city and country and do a where clause where city.CountryCode=31 and country='netherlands' :)